lib/components/ImageButton.tsx

```diff
@@ -112,8 +112,9 @@ export function ImageButton(props: Props) {
     <div
       className={classes([
         'container',
-        (buttons as boolean) ||
-          (fluid && (buttonsAlt as boolean) && 'hasButtons'),
+        fluid &&
+          ((buttons as boolean) || (buttonsAlt as boolean)) &&
```

Comment:
casting should work here?
```suggestion
          ((!!buttons as boolean) || (!!buttonsAlt as boolean)) &&
```

Reply:
Huh, yeah, thanks :D